api/file-translations/index.ts

```diff
@@ -120,4 +147,22 @@ export class SmartlingFileTranslationsApi extends SmartlingBaseApi {
         delete headers["content-type"];
         return headers;
     }
+
+    private static async downloadResponseToTranslatedFileDto(
+        response: Response): Promise<TranslatedFileDto> {
```

Comment:
Please fix formatting from:
```
(
        response: Response)
```
to:
```
(
        response: Response
)
```
or:
```
(response: Response)
```

Reply:
fixed